upstream/build.gradle

```diff
@@ -48,13 +48,23 @@ subprojects {
     buildDockerImage {
         def date = new Date().getTime().toString()
 
-        buildArgs = [
-                'SOLR_VERSION_MAJOR': "${project.solr.version.major}",
-                'SOLR_VERSION_MINOR': "${project.solr.version.minor}",
-                'SOLR_VERSION_REV'  : "${project.solr.version.rev}",
-                'SOLR_FLAVOR'       : "${project.solr.flavor}",
-                'BUILD_DATE'        : date.substring(0,date.length()-3)  // do not consider milliseconds	
-        ] << (project.hasProperty('extraBuildArgs')?project.extraBuildArgs:[:])
+        if (project.solr.version.maint)
```

Comment:
There probably is something more clean to do this

Reply:
Yes, it can be done a bit less repetitive by making use of the `<<` operator to concatenate an additional buildargs instead.

```
            buildArgs = [
                    'SOLR_VERSION_MAJOR': "${project.solr.version.major}",
                    'SOLR_VERSION_MINOR': "${project.solr.version.minor}",
                    'SOLR_VERSION_REV'  : "${project.solr.version.rev}",

                    'SOLR_FLAVOR'       : "${project.solr.flavor}",
                    'BUILD_DATE'        : date.substring(0,date.length()-3)  // do not consider milliseconds
            ] << (project.solr.version.maint?[ 'SOLR_VERSION_MAINT': "${project.solr.version.maint}"]
            << (project.hasProperty('extraBuildArgs')?project.extraBuildArgs:[:])
```